tests/conftest.py

```diff
@@ -150,6 +156,11 @@ def _bdb(_setup_database, _configure_bigchaindb):
     flush_db(conn, dbname)
 
 
+@pytest.fixture
+def _genesis(_bdb, genesis_block):
```

Comment:
I would rename this, and it's associated marker, to be `bdb_with_genesis` or similar.